### DISCUSSION TITLE
Is there a way to refresh a DAG file by dagname?

### DISCUSSION THREAD

**Discussion by appassionate** [apache/airflow#20842] [Q&A]
I imaged there have been many dag files in a directory. In airflow.cfg, we maintain the refresh interval long for uneccessary resource usage.  However, there is a need to refresh some dag file in a time and i dont want to set the "global interval". Is there any trick command or some methods i have not found?  Thanks! :  )

**Comment by potiuk** [comment]
Not for now. But in the futue it might be added.
